notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,2260 @@
+{
```

Comment:
<p>NIT: constrained</p><p> </p><p>NIT: two-dimensional sphere</p><p> </p><p>I love this explanation!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I left it as 3-dimensional sphere but 2-dimensional hypersphere... (i think that is right?)</p>

Reply:
<p>Unfortunately no! In differential geometry, we say that it is a 2-dimensional sphere or 2-dimensional hypersphere. The dimension of the sphere (or hypersphere, or manifold) is the number of degrees of freedom needed to describe the points on it. Thus, it is 2 for the sphere, which is a 2-dimensional manifold i.e. a 2-dimensional sphere.</p><p> </p><p>The 2-dimensional sphere, or 2-dimensional hypersphere is however embedded in 3-dimensional cartesian space --- thus the confusion.</p>